webapp/src/pages/Dashboard.js

```diff
@@ -104,11 +100,15 @@ const Ledger = ({ dashboard }) => {
         <Table responsive striped hover className="table-borderless">
           <thead>
             <tr>
-              <th className="d-flex justify-content-between align-items-center">
-                <span>{i18next.t("recent_ledger_lines", { ns: "dashboard" })}</span>
-                <Button variant="success" href="/funding" as={RLink}>
-                  <i className="bi bi-cash-stack me-2"></i>Add Funds
-                </Button>
+              <th>
+                <Stack direction="horizontal" gap={3}>
+                  {i18next.t("recent_ledger_lines", { ns: "dashboard" })}
```

Comment:
We need to display the "Add Funds" button in a different location in dashboard (Or ledger page?